src/ThreeEditor/Simulation/Figures/FigureScene.ts

```diff
@@ -0,0 +1,41 @@
+import * as THREE from 'three';
+import { Editor } from '../../js/Editor';
+import { generateSimulationInfo } from '../../../util/AdditionalGeometryData';
+import { BasicFigure } from './BasicFigures';
+import { SimulationPropertiesType } from '../../../types/SimProperties';
+import { UniqueChildrenNames, getNextFreeName } from '../../../util/Name/Name';
+
+export class FigureScene
+	extends THREE.Scene
+	implements SimulationPropertiesType, UniqueChildrenNames
```

Comment:
Fix to use Simulation scene

Reply:
It requires changes in many files that are already due to rewrite so will fix it later